test/System/UniqueResource.test.cpp

```diff
@@ -0,0 +1,108 @@
+#include <SFML/System/UniqueResource.hpp>
+
+#include <doctest/doctest.h>
+
+#include <SystemUtil.hpp>
+#include <type_traits>
+
+namespace
+{
+size_t deleteCount = 0;
```

Comment:
Unfortunately we decided for `std::size_t` in https://github.com/SFML/SFML/pull/2184 😛